schutzbot/Jenkinsfile

```diff
@@ -814,6 +810,11 @@ void run_tests(test_type) {
     }
 
     if (test_type == 'integration') {
+        sh (
+            label: "Regression tests",
+            script: "/usr/libexec/tests/osbuild-composer/regression.sh"
+        )
+
```

Comment:
Doesn't matter much in which group this script is executed but ATM the integration group is rather big so you may leave this under the base group.